changelog.txt

```diff
@@ -1,3 +1,17 @@
+4.14.0
+============
+New Features: 
+
+**MSAL no longer calls the OIDC endpoint, as it can infer the authorization and token endpoints based on the authority URL. This will speed up token acquisition, especially for multi-tenant applications, as fewer network calls will be made. For details see [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1750)
```

Comment:
I'm not sure we can write "MSAL no longer calls the OIDC endpoint". Do we mean the OIDC introspection endpoint? or the OIDC metadata endpoint?